pcrf/include/session.h

```diff
@@ -248,6 +296,116 @@ class GxSessionKey
    std::string m_apn;
 };
 
+class GxSessionState;
+
+class GxSessionProc
```

Comment:
add string name for current name of procedure
and also add the deletion of previous procedure in setCurrentProcedure function